### PR TITLE
docs: improve documentation verbosity

### PR DESCRIPTION
**Pull request by codebytere**
Add a little more information about some of the command-specific documention and clean up examples. 

cc @ckerr @MarshallOfSound

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -258,36 +278,52 @@ e test --runners=main
...
 
-Valid patch directories include:
+Valid patch directories can include:
 
 * `node`
 * `v8`
 * `boringssl`
 * `chromium`
+* `perfetto`
+* `icu`
```
Comment (ckerr) [inline README.md]: Are these two correct? I don't see them in https://github.com/electron/electron/blob/master/patches/config.json

File: src/e-build.js [hunk 2]
```diff
@@ -24,11 +24,11 @@ function runGNGen(config) {
...
-  const argsFile = path.resolve(evmConfig.outDir(config), 'args.gn');
-  if (!fs.existsSync(argsFile)) return runGNGen(config);
-  const contents = fs.readFileSync(argsFile, 'utf8');
-  // If the current args do not match the args file, re-run gen
-  if (contents.trim() !== config.gen.args.join('\n').trim()) return runGNGen(config);
+  // const argsFile = path.resolve(evmConfig.outDir(config), 'args.gn');
+  // if (!fs.existsSync(argsFile)) return runGNGen(config);
+  // const contents = fs.readFileSync(argsFile, 'utf8');
+  // // If the current args do not match the args file, re-run gen
+  // if (contents.trim() !== config.gen.args.join('\n').trim()) return runGNGen(config);
```
Comment (ckerr) [inline src/e-build.js]: Is this part intentional for a docs PR?

File: README.md (hunk 1 above)
Comment (codebytere) [reply README.md]: @ckerr i included them as they exist on some older branches (see: https://github.com/electron/electron/pull/21969); our patches can run a range but i'd rather have them here for verbosity's sake so folks know they're a possibility

File: src/e-build.js (hunk 2 above)
Comment (codebytere) [reply src/e-build.js]: ope

no

**Pull request merged**